Client/Frontend/Fakespot/Views/FakespotAdView.swift

```diff
@@ -212,7 +210,7 @@ class FakespotAdView: UIView, Notifiable, ThemeApplicable, UITextViewDelegate {
             secondRowStackView.axis = .horizontal
             spacer.isHidden = true
         }
-        setupLayout()
+        setupSubviews()
```

Comment:
`setupLayout` is the name for method that creates the layout in most new code. I would say it's a pattern throughout the app by now. So we should stick to that pattern here as well.